measurements/perplexity/requirements.txt

```
@@ -0,0 +1,6 @@
+# TODO: fix github to release
+git+https://github.com/huggingface/evaluate.git@b6e6ed7f3e6844b297bff1b43a1b4be0709b9671
```

I think this was also the reason the other space was broken. I'll fix this to the actual release for all spaces next week. 
```suggestion
git+https://github.com/huggingface/evaluate.git@505123230059f9605da8951880eddc9d1fbf4278
```